content/ways-to-train.md

```diff
@@ -42,208 +39,89 @@ keywords:
 
-The Teach First Leadership Development Programme is a two-year salaried programme.
+Although you often have to pay course fees of around £9,250 you can get [funding that you do not have to pay back](/funding-your-training) for certain subjects.
 
-It combines leadership development with teacher training, and focuses on training exceptional teachers to teach in challenging schools.
+You can also apply for a [student loan](/funding-your-training).
 
-You’ll learn on the job while you work towards Qualified Teacher Status (QTS) and a Postgraduate Diploma in Education (PGDE).
+Some training programmes offer a salary, although these can be harder to find.
 
-A PGDE contains up to 120 credits at master’s degree level.
+<a class="button" href="https://www.find-postgraduate-teacher-training.service.gov.uk/">Find courses near you</a>
 
-<p class="call-to-action__action">
-  <a href="https://www.teachfirst.org.uk/">Find a Teach First <span>course</span></a>
-</p>
+#### Teach First
 
-$undergraduate$
+You can also consider [Teach First](https://www.teachfirst.org.uk/). It's a salaried, 2-year programme leading to a PGDE and qualified teacher status.
 
- You can get Qualified Teacher Status (QTS) as part of an undergraduate degree course.
- 
- This could be right for you if you do not have a degree yet.
- 
- You can also [get funding](/funding-your-training).
- 
- <p class="call-to-action__action">
-   <a href="https://digital.ucas.com/search">Find an undergraduate <span>course</span></a>
- </p>
+### If you do not have a degree
 
-$assessment-only$
+You can get qualified teacher status as part of an undergraduate degree.
 
-If you’re an experienced teacher with a degree, you can gain QTS without doing a teacher training programme.
+Qualifications vary depending on the course. For example, you could get qualified teacher status with a BA honours or a BSc honours.
 
-You’ll need to show evidence that you meet the standards for QTS.
+Courses typically cost £9,250 per year and can take up to 4 years but you can [get funding](https://www.gov.uk/student-finance-calculator).
 
-A number of universities and School Centred Initial Teacher Training programmes (SCITTs) offer assessment only routes. Their criteria for entry may vary.
+<a class="button" href="https://digital.ucas.com/search">Search for an undergraduate course</a>
 
-You can apply to them directly.
+### If you already have teaching experience
 
-<p class="call-to-action__action">
-  <a href="/assessment-only-providers">See assessment only <span>providers</span></a>
-</p>  
+If you have a degree and you're an experienced teacher, you may be able to get qualified teacher status through a 12-week 'assessment only' programme.
 
-## Specialised routes into teaching
+[Apply directly to assessment only teacher training providers](/assessment-only-providers) and ask them about entry criteria.
 
-### Career changers
+### If you're a career changer
```

Comment:
Is being a career changer an internal phrase, or a generally understood one? I hadn't heard it before. Slight nervousness on labelling people rather than talking about their choices? e.g. Other options don't name that person as 'A teacher already' vs 'if you already have teaching experience'